e2e/ansible/playbooks/utils/delete_namespaces.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: Deletion of test specific  namespace.
+  shell: source ~/.profile; kubectl delete ns {{ namespace }}
+  args:
+    executable: /bin/bash
+  delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+
+- name: Check the namespace status.
+  shell: source ~/.profile; kubectl get ns 
+  args:
+    executable: /bin/bash
+  register: np
+  delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+  until: "'{{ namespace }}' not in np.stdout"
+  delay: 30
+  retries: 15
+  ignore_erros: true
```

Comment:
Typo at "ignore_erros" ! Also, why are we ignoring this ? The idea is to make an effort at all costs. 

Also, the namespace should ideally be passed as a param to util as a best practice (though it should work even w/o by virtue of adding vars file)